source/agora/node/FullNode.d

```diff
@@ -733,4 +737,29 @@ public class FullNode : API
             });
         }
     }
+
+    /***************************************************************************
+
+        Params:
+            tx_hashes = A Set of Transaction hashes
+
+        Returns:
+            Transactions corresponding to the requested hashes or
+            Transaction.init for hashes that can't be found in the pool
```

Comment:
That's not what the code does though, it doesn't return `Transaction.init` but just skips those transactions entirely.

Reply:
Oops, copy paste error.